server/errors.go

```diff
@@ -0,0 +1,13 @@
+package server
+
+type nonRetryableTranscodeError struct {
```

Comment:
Most of this file could be omitted if we embed `error` in the struct i.e.

```
type nonRetryableTranscodeError struct {
    error
}
```

Reply:
Is this file still needed? I don't see `nonRetryableTranscodeError` used anywhere else in this PR